Use PHP 5.5 syntax.

Arantor opened the pull request:
This should fix the only obvious issues in Slim-Bridge that prevent it working on PHP 5.5. Fixes #8 

I have only given it basic testing, it seems to work for my colleagues (I don't use it myself)

Arantor committed:
Use PHP 5.5 syntax.

This should fix the only obvious issues in Slim-Bridge that prevent it working on PHP 5.5. Fixes #8

mnapoli commented:
Thank you, could you update `.travis.yml` so that tests runs on PHP 5.5?

mnapoli commented on composer.json:
```diff
@@ -14,7 +14,7 @@
     },
     "require": {
-        "php": "~5.6|~7.0",
+        "php": ">=5.5.0",
```
Could you keep the `~5.5|~7.0` notation? That avoids speculating on BC breaks in PHP 8

Arantor committed:
Update Travis configuration as requested, though I've never used Travis before so this might be wrong.

Arantor committed:
Changed Composer syntax as requested.

Arantor commented:
I've updated it - though I've never used Travis so I don't know if that's the correct way to do it.

mnapoli commented:
Travis configuration is correct, tests are failing because on invalid PHP code in `config.php`: https://travis-ci.org/PHP-DI/Slim-Bridge/jobs/141813495#L265

Also it seems that PHPUnit's version isn't compatible with PHP 5.5: https://travis-ci.org/PHP-DI/Slim-Bridge/jobs/141813494#L156 (you might need to change PHPUnit's version)

Arantor committed:
Fix syntax error from enthusiastic bulk find/replace.

(For reasons I don't understand, the version we had at work was different to the version in GitHub, I just applied the same find/replace)

Arantor commented:
Ugh. The check it fails is that this all hinges on PHPUnit, which also requires 5.6, and I doubt you'd merge a patch in that would cause the build to always fail as far as Travis is concerned.

Guess this was a waste of time :disappointed: but I'll leave the pull request up in case anyone who wants PHP 5.5 support can somehow make use of it. We're not all in the happy position to be using current PHP versions, unfortunately.

mnapoli commented:
@Arantor you can change PHPUnit's version (use PHPUnit 4.*) so that it works with PHP 5.5

mnapoli commented:
Replaced by #14
